treeherder/log_parser/management/commands/store_error_summary.py

```diff
@@ -0,0 +1,40 @@
+from cStringIO import StringIO
+
+import requests
+from django.core.management.base import BaseCommand, CommandError
+from mozlog import reader
+
+from treeherder.model.derived import JobsModel
+from treeherder.model.models import Failure, Repository
+
+
+class Command(BaseCommand):
+    args = '<log_url>, <job_guid>, <repository>'
+    help = 'Download, parse and store the given failure summary log.'
```

Comment:
We currently have a task to download and ingest a log (https://github.com/mozilla/treeherder/blob/master/treeherder/etl/management/commands/test_parse_log.py) - is it worth bringing that over into the new treeherder/log_parser/management/commands/ directory too?